src/app/object-schema/object-schema.component.ts

```diff
@@ -0,0 +1,61 @@
+import { Component, OnInit, Input } from '@angular/core';
+import { ObjectSchemaFormControl } from '../formControls/ObjectSchemaFormControl';
+import { ObjectSchemaEditorService } from '../services/object-schema-editor/object-schema-editor.service';
+
+@Component({
+  selector: 'object-schema',
+  templateUrl: './object-schema.component.html',
+  styleUrls: ['./object-schema.component.scss']
+})
+export class ObjectSchemaComponent implements OnInit {
+
+  private maxLevel: number = 5; 
+  @Input() public field!: ObjectSchemaFormControl;
```

Comment:
Minor style correction but I like to see the @Input directives at the top:

Inputs
Outputs
Public
Private

ctor()..

Reply:
Ok, thanks for letting me know - fixed it